_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+
+While this does not directly document CO<sub>2</sub> emissions, we can still estimate them from the available data.  We know how much heat is released from burning fossil fuels at the plant on a monthly basis, in millions of BTUs (MMBTU).  This information can be multiplied by [emissions factors from the US EPA](https://www.epa.gov/sites/production/files/2018-03/documents/emission-factors_mar_2018_0.pdf){:target="_blank"} that are estimates of how many kilograms of CO<sub>2</sub> are emitted for every MMBTU of combusted fuel.  This step tells us how many kilograms of CO<sub>2</sub> are emitted on a monthly basis.  By dividing this number of the amount of electricity generation, we then get the CO<sub>2</sub> emissions intensity in the form of $$\frac{kg\ CO_2}{MWh}$$.
+
+In the plot below, we use the same approach as that in the sulfur content plot above:
+
+* Yearly generation is sorted from most CO<sub>2</sub> intensive (old coal plants) to the least intensive (renewables)
+* Using this sorting, the cumulative total generation is calculated.
+* The area under the curve represents the total CO<sub>2</sub> emissions for that year.
+
+Here 2001 is used as the reference year.  Vertical dashed lines are added to indicate total generation for that year, as nuclear and renewables have zero emissions and their generation contributions would not be visible otherwise on the plot.  Also, the y axis is clipped at 1500 kg CO<sub>2</sub>/MWh to reduce the vertical scale shown.  The plants with higher values can be older less efficient power plants, or plants that have been completely shut down and need to consume extra fuel to bring the equipment back up to operating temperatures.
+
+![CO2 Intensity]({{ site.baseurl }}public/images/YearlyEmissionsGenerationProfile.gif)
+<sup>_Yearly profiles of US electricity generation by carbon intensity._</sup>
+
+From the plot we can see that the amount of generation peaked around 2007 and has been roughly stable since then.  While some increases in total emissions occurred after 2001, by looking at 2016, we see that generation from fossil fuels is at the same level as it was in 2001. Furthermore, we can see two horizontal "shelves", with the lower one around 400 kg CO<sub>2</sub>/MWh corresponding to generation from natural gas, with the upper one at 900 kg CO<sub>2</sub>/MWh corresponding to generation from coal[^Not_Life_Cycle_Emissions_Intensity].  In 2016, these shelves are quite visible, and the light gray area represents a large amount of emissions that were reduced by switching from coal to natural gas.  Overall it's clear that the US has been steadily decarbonizing the electricity sector.
```

Comment:
--> "We can also see two horizontal "shelves", with the lower one around 400 kg CO<sub>2</sub>/MWh corresponding to generation from natural gas, and the upper one at 900 kg CO<sub>2</sub>/MWh corresponding to generation from coal[^Not_Life_Cycle_Emissions_Intensity]. "